### PR TITLE
Add ciocheck recipe

### PR DESCRIPTION
**Pull request by goanpeca**

### CODE REVIEW

**Comment by goanpeca** [comment]
@jakirkham merge?

File: recipes/ciocheck/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+test:
+  imports:
+    - ciocheck
+
+about:
+  home: https://github.com/ContinuumIO/ciocheck
+  summary: "Continuum Analytics linter/formater/tester helper"
+  license: MIT
+  license_family: MIT
```
Comment (synapticarbors) [inline recipes/ciocheck/meta.yaml]: Add:
- `license_file: LICENSE.txt`
- `dev_url: https://github.com/ContinuumIO/ciocheck`

File: recipes/ciocheck/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{% set name = "ciocheck" %}
+{% set version = "0.1.1" %}
+{% set hash_type = "sha1" %}
```
Comment (synapticarbors) [inline recipes/ciocheck/meta.yaml]: `sha256` is preferred

File: recipes/ciocheck/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+{% set hash = "8cfedcc1d462f6f55ab33d2ea08b91ad55dca6e9" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_rev: v0.1.1
+  git_url: https://github.com/ContinuumIO/ciocheck.git
```
Comment (synapticarbors) [inline recipes/ciocheck/meta.yaml]: I'm not sure what the policy is (if there even is one), but I feel like most packages either pull from pypi.io or the the github release archive (https://github.com/ContinuumIO/ciocheck/archive/v0.1.1.tar.gz).

Comment (jakirkham) [inline recipes/ciocheck/meta.yaml]: That is the policy. 😄 Unless of course there is a good reason not to use some form of tarball.

File: recipes/ciocheck/meta.yaml (hunk 2 above)
Comment (goanpeca) [inline recipes/ciocheck/meta.yaml]: How do I get that from https://github.com/ContinuumIO/ciocheck/archive/v0.1.1.tar.gz ?

File: recipes/ciocheck/meta.yaml (hunk 1 above)
Comment (goanpeca) [inline recipes/ciocheck/meta.yaml]: donde

File: recipes/ciocheck/meta.yaml (hunk 2 above)
Comment (goanpeca) [inline recipes/ciocheck/meta.yaml]: Nevermind!

**Comment by goanpeca** [comment]
@synapticarbors, @jakirkham, all good for merging? I updated with the suggestions :-)

**Pull request merged**